cli/root.go

```diff
@@ -175,6 +181,10 @@ func (r *RootCmd) RunWithSubcommands(subcommands []*serpent.Command) {
 			//nolint:revive,gocritic
 			os.Exit(code)
 		}
+		if errors.Is(err, ErrSilent) {
```

Comment:
`&exitError{code: 1}` could potentially be used instead, but I see other uses of it have disappeared from the code base.